webapp-django/crashstats/settings/base.py

```diff
@@ -628,21 +623,6 @@ def path(*dirs):
     default=60 * 2
 )
 
-# URL to send the Google Analytics pageviews and event tracking.
-# The value of this is extremely unlikely to change any time soon,
-GOOGLE_ANALYTICS_API_URL = config(
-    'GOOGLE_ANALYTICS_API_URL',
-    'https://ssl.google-analytics.com/collect'
-)
-
-# If calls to the Google Analytics API are done asynchronously, this
-# value can be quite high (5-10 seconds).
-GOOGLE_ANALYTICS_API_TIMEOUT = config(
-    'GOOGLE_ANALYTICS_API_TIMEOUT',
-    5,  # seconds
-    cast=int
-)
-
```

Comment:
I think this is my fault. I ripped out a bunch of server-side GA code and I think I left this in by accident.